slurm/slurm_meter

```diff
@@ -214,6 +215,9 @@ class SlurmMeter(SlurmProbe):
         # Loop over completed jobs
         time_end = None
         server_id = self.get_db_server_id()
+        # don't consider jobs that have ended within the last 15 min
+        # https://opensciencegrid.atlassian.net/browse/SOFTWARE-3347
+        cutoff_ts = min(self.checkpoint.val, int(time.time()) - 15 * 60)
         for job in self.sacct.completed_jobs(self.checkpoint.val):
```

Comment:
Don't you have to send `cutoff_ts` to the complete_jobs function?

Reply:
Yeah derp.  I realized this last night in my sleep :D